app/controllers/application_controller.rb

```diff
@@ -4,6 +4,10 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery unless: -> { request.format.json? }, with: :exception
 
+  def default
+    render text: 'ok'
```

Comment:
added a test and a comment to app controller --- not really needed per se but matches what some other API level apps we built do (and otherwise you get the generic Rails "welcome to your app" message if you happen to hit the root)